src/NzbDrone.Core/Configuration/ConfigFileProvider.cs

```diff
@@ -216,6 +223,36 @@ public string UrlBase
             }
         }
 
+        public UseProxy ProxyEnabled
+        {
+            get { return GetValueEnum<UseProxy>("ProxyEnabled", UseProxy.Off); }
+        }
+
+        public ProxyType ProxyType
+        {
+            get { return GetValueEnum<ProxyType>("ProxyType", ProxyType.Http); }
+        }
+
+        public string ProxyHostname
+        {
+            get { return GetValue("ProxyHostname", "localhost"); }
```

Comment:
We are not getting at the config in Common (its in Core via ProxyHttpInterceptor.cs). 

I did originally do it all via the ConfigService/the db. But I moved it to the ConfigFileService because everything else on that  generals settings screen used that one, and I couldn't work out how to split the settings into two so they get persisted to different configuration services.